button/button.css

```diff
@@ -0,0 +1,91 @@
+@import"../for-import/reset.css";
+@import"../for-import/colors.css";
+
+section {
```

Comment:
avoid grabbing elements in a component library, try using classes.

Reply:
That was a good one. I will fix it soon.